### PR TITLE
Feature/sidebar horizontal

### PR DESCRIPTION
**Pull request by huitre**
This pull request allow a user to display the sidebar in a horizontal way instead of the current behavior, which is vertical, by passing sidebarHorizontal props to the Editor.

We use this behavior in combination with my other PR #151, sidebar visible only on empty block,  on our current project at Prisma Media.

Hope theses PR are useful for others !

### CODE REVIEW

File: src/components/MegadraftEditor.js [hunk 1]
```diff
@@ -320,6 +320,7 @@ export default class MegadraftEditor extends Component {
             onChange: this.onChange,
             maxSidebarButtons: this.props.maxSidebarButtons,
             modalOptions: this.props.modalOptions,
+            sidebarHorizontal: this.props.sidebarHorizontal,
```
Comment (ricobl) [inline src/components/MegadraftEditor.js]: Can you please rename it to `horizontalSidebar`?

File: src/styles/sidebar.scss [hunk 2]
```diff
@@ -69,9 +70,9 @@
     &__items {
         list-style: none;
         padding: 0;
-        margin-top: 8px;
-        padding-top: 2px;
-        transition: max-height 0.5s ease;
+        margin-top: 0px;
+        padding-top: 4px;
```
Comment (ricobl) [inline src/styles/sidebar.scss]: Changing the `margin` and `padding` here will change the spacing in the vertical sidebar. Please keep the same aspect.

**Comment by nihaux** [comment]
@huitre hey ! any news on that ? :D

**Comment by marcelometal** [comment]
@huitre Hi! Any news on that?

**Pull request closed without merging**